packages/jellyfish-api-core/__tests__/category/loan/createLoanScheme.test.ts

```diff
@@ -0,0 +1,106 @@
+import { LoanMasterNodeRegTestContainer } from './loan_container'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { UTXO } from '@defichain/jellyfish-api-core/category/oracle'
+import BigNumber from 'bignumber.js'
+
+describe('Loan', () => {
+  const container = new LoanMasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
```

Comment:
Consider using the new jellyfish-testing framework from now on

Reply:
Scheme related rpcs are the most fundamental rpcs for loan. We don't use a group of rpcs to setup something.
Nevertheless, I am going to do so in the vault or auction related rpcs because creating a vault or auction require scheme, loanToken, collateralToken rpcs to be setup.

Reply:
You can use `testing.rpc` instead of `client` and `testing.container` instead of `container`.
We need to standardize testing with the new framework.